### PR TITLE
etcd quorum guard

### PR DESCRIPTION
**Pull request by RobertKrawitz**
Deployment and disruption budget to ensure that no more than 1 etcd at a time may be unhealthy/dead.

### CODE REVIEW

File: ci-operator/config/openshift/etcd-quorum-guard/openshift-etcd-quorum-guard-master.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      cpu: 100m
+      memory: 200Mi
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+tests:
+- as: unit
```
Comment (sjenning) [inline ci-operator/config/openshift/etcd-quorum-guard/openshift-etcd-quorum-guard-master.yaml]: we don't have a Makefile in this repo yet so remove the `unit` and `verify` test blocks

Comment (RobertKrawitz) [reply ci-operator/config/openshift/etcd-quorum-guard/openshift-etcd-quorum-guard-master.yaml]: In which repo -- `release` or `etcd-quorum-guard`?  I added one to `etcd-quorum-guard` in the last PR.

Comment (sjenning) [reply ci-operator/config/openshift/etcd-quorum-guard/openshift-etcd-quorum-guard-master.yaml]: https://github.com/openshift/etcd-quorum-guard/blob/master/Makefile

I see the `verify` target but not the `unit` target

Comment (sjenning) [reply ci-operator/config/openshift/etcd-quorum-guard/openshift-etcd-quorum-guard-master.yaml]: Sorry, nm, the target is `test` :+1:

**Comment by sjenning** [comment]
LGTM
@petr-muller could you re-add lgtm?  I think @RobertKrawitz fixed all the test issues.

**Pull request merged**